diesel_tests/Cargo.toml

```diff
@@ -12,13 +12,14 @@ dotenv = ">=0.8, <0.11"
 [dependencies]
 assert_matches = "1.0.1"
 chrono = { version = "0.3" }
-diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuid", "serde_json", "network-address"] }
+diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuid", "serde_json", "network-address", "numeric"] }
 diesel_codegen = { path = "../diesel_codegen" }
 dotenv = ">=0.8, <0.11"
 quickcheck = { version = "0.3.1", features = ["unstable"] }
 uuid = { version = ">=0.2.0, <0.6.0" }
 serde_json = { version=">=0.9, <2.0" }
 ipnetwork = "0.12.2"
+bigdecimal = { version = ">=0.0.7" }
```

Comment:
Mmm. What about "0.0"? BigDecimal will probably release some more minor additions before it's 1.0; and it'll be pretty tedious to bump the version every time.